### PR TITLE
[RFC][UN-6730] New design changes

### PR DESCRIPTION
**Pull request by mmelo**
Refers to: https://uniplaces.atlassian.net/browse/UN-6730

**Warning**: be mindful that this version will break typography on all platforms right at a build level. Thus making changes on every platforms that update to this version is imperative.

### CODE REVIEW

File: src/styles/_common/_typography.scss [hunk 1]
```diff
@@ -1,130 +1,69 @@
...
-    @include font("IntervalNextReg");
-    font-size: em(24px);
+    font-size: $headline;
+    color: color('text','title');
+
+    &--light {
+        @include font("IntervalNextLi");
+    }
 }
 @mixin title() {
```
Comment (niceoutput) [inline src/styles/_common/_typography.scss]: Give some space between mixins

File: src/styles/_common/_typography.scss [hunk 2]
```diff
@@ -1,130 +1,69 @@
...
 }
-@mixin label-title() {
-    @include font("IntervalNextBook");
-    font-size: em(18px);
+@mixin subheader() {
+    @include font("IntervalNextReg");
+    font-size: $subheader;
+    color: color('text','subheader');
 }
 @mixin subtitle() {
```
Comment (niceoutput) [inline src/styles/_common/_typography.scss]: space between mixins..

File: src/styles/_common/_typography.scss [hunk 3]
```diff
@@ -1,130 +1,69 @@
...
-    line-height: 22px;
-    color: color('text','subtitle');
-
-    &--strong {
-        color: color('text','title');
-    }
+    color: color('text', 'subheader');
+    opacity: .7;//jesus Xavier... really?
 }
 @mixin body() {
```
Comment (niceoutput) [inline src/styles/_common/_typography.scss]: The same as above

File: src/styles/_common/_typography.scss [hunk 4]
```diff
@@ -1,130 +1,69 @@
...
     @include font("IntervalNextBook");
     font-size: $body;
-    line-height: 21px; 
-    color: color('text','body');
+    color: color('text', 'body');
+    line-height: 21px;
+}
+@mixin caption() {
```
Comment (niceoutput) [inline src/styles/_common/_typography.scss]: The same as above

File: src/styles/_common/_typography.scss [hunk 5]
```diff
@@ -1,130 +1,69 @@
...
+    @include font("IntervalNextBook");
+    font-size: $caption;
+}
+@mixin small() {
```
Comment (niceoutput) [inline src/styles/_common/_typography.scss]: The same as above

File: src/styles/_common/_typography.scss (hunk 1 above)
Comment (asantos00) [inline src/styles/_common/_typography.scss]: I think they're displayed this way to "group" them and separate from the "fonts" mixins, but not sure

File: src/styles/_common/_typography.scss [hunk 6]
```diff
@@ -1,130 +1,69 @@
...
+    @include font("IntervalNextBook");
+    font-size: $small;
 }
 @mixin anchor() {
```
Comment (niceoutput) [inline src/styles/_common/_typography.scss]: The same as above

File: src/styles/_common/_typography.scss (hunk 1 above)
Comment (mmelo) [inline src/styles/_common/_typography.scss]: that was the rationale but indded I must follow coding styles.

File: src/styles/_components/_image-gallery.scss [hunk 7]
```diff
@@ -40,7 +40,8 @@
 
         &__message {
             width: 100%;
-            font-size: em(20px);
+            font-size: em(16px);
+            opacity: 0.7;//jesus Xavier, really?
```
Comment (niceoutput) [inline src/styles/_components/_image-gallery.scss]: Like the acronim of xavier....Jesus... ( remove this comment please )

**Pull request merged**